configs/templates/_do.txt

```diff
@@ -83,22 +81,78 @@ SECURITY_GROUPS = not_yet_applicable
 [AI_TEMPLATES : HADOOP ]
 HADOOP_HOME = /usr/local/hadoop
 
-# DigitalOcean has made public images available for the following 3 workloads:
 [VM_TEMPLATES : DO_CLOUDCONFIG ]
-CHECK = size:512mb, imageids:1, imageid1:to_replace
-# These are public DigitalOcean images accessible by anyone with a valid DigitalOcean account.
 # imageid can be either the actual ID number or more easily the name in the case of a private account
-TINYVM = size:512mb, imageids:1, imageid1:cloudbench-nullworkload-on-1604.060117-1
-NETCLIENT = size:512mb, imageids:1, imageid1:cloudbench-nullworkload-on-1604.060117-1
-NETSERVER = size:512mb, imageids:1, imageid1:cloudbench-nullworkload-on-1604.060117-1
-YATINYVM = size:512mb, imageids:1, imageid1:cloudbench-nullworkload-on-1604.060117-1
+# The images named cloudbench-*-on-1604.060117-1 are accessible by anyone with a valid DigitalOcean account
+ACMEAIR = size:4gb, imageid1:cb_acmeair
+APACHE = size:4gb, imageid1:cb_wrk
+BONNIE = size:2gb, imageid1:cb_bonnie
+BTEST = size:2gb, imageid1:cb_btest
+CAFFE = size:4gb, imageid1:cb_caffe
 CASSANDRA = size:4gb, imageids:1, imageid1:cloudbench-ycsb-cassandra-on-1604.060117-1
-YCSB = size:4gb, imageids:1, imageid1:cloudbench-ycsb-cassandra-on-1604.060117-1
-SEED = size:4gb, imageids:1, imageid1:cloudbench-ycsb-cassandra-on-1604.060117-1
+CHECK = size:512mb, imageids:1, imageid1:to_replace
+CLIENT_IBM_DAYTRADER = size:4gb, imageid1:cb_daytrader
```

Comment:
Except for the `check` image, can we revert this change? These are not public images. 

If the user runs the workload builder (and we should encourage them to do so), then we might want to provide an "include" file after the builder is done that shows them how to define the new workloads.

But, otherwise, I would prefer we did not have workload definitions with images that don't really exist in the public cloud.